js/client/modules/@arangodb/arango-database.js

```diff
@@ -1232,7 +1216,7 @@ ArangoDatabase.prototype._createView = function (name, type, properties) {
   var nname = requestResult.name;
 
   if (nname !== undefined) {
-    this._registerView(nname, new this._viewConstructor(this, requestResult));
+    this[nname] = new ArangoView(this, requestResult);
```

Comment:
I know that this is not your fault, but what is `nname`. Not clear without reading the whole code. Maybe we could change that.

Reply:
Ok, fixing it...